spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -603,13 +617,10 @@ public class ExampleItem {
 }
 ----
 
-===== Read with offsets and limits
-
-This feature is not supported yet.
+===== Read with offsets, limits, and sorting
 
-===== Sorting
-
-This feature is not supported yet.
+`DatastoreRepository` and custom defined entitiy repositories implement Spring Data's `PagingAndSortingRepository`, which supports offsets and limits using page numbers and page sizes.
+Paging and sorting options are also supported in `DatastoreTemplate` by supplying a `DatastoreQueryOptions` to `findAll`.
```

Comment:
```suggestion
Paging and sorting options are also supported in the `DatastoreTemplate` by supplying a `DatastoreQueryOptions` to `findAll`.
```

Reply:
I always think of the class, not an instance when referencing. "The DatastoreTemplate [class]" sounds ok to me.

Reply:
However "class" isn't here, and I don't think it's necessary to say that it is a class. 

I've been most comfortable and I think it's most natural to refer to classes without "the". For example when we talk about `Object`, `String`, or `Long`, we never say "the String class".